schema/mapml.sch

```diff
@@ -1,115 +1,113 @@
-            <sch:assert test="exists(//datalist[@id eq $listid])">A datalist must be associated to a shard-type input</sch:assert>
+            <sch:assert test="exists(//h:datalist[@id eq $listid])">A datalist must be associated to a shard-type input</sch:assert>
             <sch:report test="exists(./@value)">A shard-type input must not have a @value</sch:report>
         </sch:rule>
-        <sch:rule context="input[@shard]">
+        <sch:rule context="h:input[@shard]">
             <sch:assert test=".[@type eq 'hidden']">A shard-type input must have @type="hidden"</sch:assert>
         </sch:rule>
-        <sch:rule context="extent">
-            <sch:assert test="input[@type eq 'zoom']">Extent must have a zoom input</sch:assert>
-            <sch:assert test="(count((input[@type='location'][@axis eq 'i'] union input[@type='location'][@axis eq 'j'])) mod 2) eq 0">location inputs with axis=i or j must come in pairs</sch:assert>
-            <sch:assert test="(count(input[@type='location'][@axis eq 'easting' or @axis eq 'northing']) mod 2) eq 0">location inputs with axis=easting or northing must come in pairs</sch:assert>
-            <sch:assert test="(count(input[@type='location'][@axis eq 'latitude' or @axis eq 'longitude']) mod 2) eq 0">location inputs with axis=latitude or longitude must come in pairs</sch:assert>
-            <sch:assert test="(count(input[@type='location'][@axis eq 'row' or @axis eq 'column']) mod 2) eq 0">location inputs with axis=row or column must come in pairs</sch:assert>
-            <sch:assert test="(count(input[@type='location'][@axis eq 'x' or @axis eq 'y']) mod 2) eq 0">location inputs with axis=x or y must come in pairs</sch:assert>
+        <sch:rule context="h:extent">
+            <sch:assert test="h:input[@type eq 'zoom']">Extent must have a zoom input</sch:assert>
+            <sch:assert test="(count((h:input[@type='location'][@axis eq 'i'] union h:input[@type='location'][@axis eq 'j'])) mod 2) eq 0">location inputs with axis=i or j must come in pairs</sch:assert>
+            <sch:assert test="(count(h:input[@type='location'][@axis eq 'easting' or @axis eq 'northing']) mod 2) eq 0">location inputs with axis=easting or northing must come in pairs</sch:assert>
+            <sch:assert test="(count(h:input[@type='location'][@axis eq 'latitude' or @axis eq 'longitude']) mod 2) eq 0">location inputs with axis=latitude or longitude must come in pairs</sch:assert>
+            <sch:assert test="(count(h:input[@type='location'][@axis eq 'row' or @axis eq 'column']) mod 2) eq 0">location inputs with axis=row or column must come in pairs</sch:assert>
+            <sch:assert test="(count(h:input[@type='location'][@axis eq 'x' or @axis eq 'y']) mod 2) eq 0">location inputs with axis=x or y must come in pairs</sch:assert>
             <sch:report test="(exists(@method) and not(exists(@action)))">An extent with a method attribute must have an action attribute</sch:report>
```

Comment:
I believe this line can be removed. Anyhow, I have not familiarized myself with neither rnc nor sch, so feel free to merge. 👍

Reply:
To clarify I'm referring to line 34 for the deprecated method / action attributes.